hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/EventCondition.java

```diff
@@ -100,6 +118,7 @@ public boolean match(Event value) {
     private static final String ID = "id";
     private static final String CTIME = "ctime";
     private static final String TEXT = "text";
+    private static final String CATEGORY = "category";
     private static final String CONTEXT = "context.";
```

Comment:
This is a mixed feeling to me.
As we have two types of inference here:
- Against data, where the data is explicit.
- Against Events, where we have only a text, but potentially possibility to "draw" a complex type using the context.
So, for events, I understand that context could be part of the "first class" data where an user is interested on inference.
I don't know which could be better, I vote to leave it as it seems events have its own scenario.
Perhaps we can talk more about this.